apidocs/cloud-api-source/source/includes/action.md

```diff
@@ -13,9 +13,9 @@
     "can_be_canceled": false,
     "location": "New York, USA",
     "method": "POST",
-    "object": "/api/infra/v1/cluster/eea638f4-b77a-4183-b241-22dbd7866f22/",
-    "path": "/api/infra/v1/cluster/",
-    "resource_uri": "/api/audit/v1/action/6246c558-976c-4df6-ba60-eb1a344a17af/",
+    "object": "/api/infra/v1/user_namespace/cluster/eea638f4-b77a-4183-b241-22dbd7866f22/",
+    "path": "/api/infra/v1/user_namespace/cluster/",
+    "resource_uri": "/api/audit/v1/user_namespace/action/6246c558-976c-4df6-ba60-eb1a344a17af/",
```

Comment:
Fixed, and added `user` to the actions api reference, as we discussed in Slack. :)